source/renderer/app/config/votingConfig.js

```diff
@@ -19,3 +20,4 @@ export const VOTING_REGISTRATION_CAST_END_DATE = new Date(
 export const VOTING_REGISTRATION_NEW_START_DATE = new Date(
   'Nov 11, 2021, 11:00 UTC'
 );
+export const VOTING_REWARD = 1040000;
```

Comment:
@lucas-barros please add a comment at the end of this line and specify the unit value - e.g. see how it was done for the other variables:

```
export const VOTING_REGISTRATION_END_CHECK_INTERVAL = 3000; // 3 seconds | unit: milliseconds
```

Reply:
Updated, thanks!